app/models/interaction.rb

```diff
@@ -0,0 +1,15 @@
+class Interaction < ApplicationRecord
```

Comment:
I'm not about the purpose of this model, should we need it? why cannot we query it directly from the phone call and probably message(sms) in the future?